llvm/test/CodeGen/RISCV/skip-mir-comment-trailing-whitespace.mir

```diff
@@ -0,0 +1,28 @@
+# NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+# RUN: llc %s -mtriple=riscv64 -mattr=+v | FileCheck %s
+
+--- |
```

Comment:
Don't need IR section

Reply:
How to remove IR while having `store unknown-size into %ir.base`?

Reply:
Remove `into %ir.base`